data/outersys.ssc

```diff
@@ -633,15 +651,21 @@ ReferencePoint "Varda-Ilmare" "Sol"
 		ArgOfPericenter	346.87585252865
 		MeanAnomaly	344.208559857227
 	}
+	BodyFrame	{ EquatorJ2000 {} }
 	UniformRotation
 	{
 		Period	11.05
+		Inclination	133.9981188
+		AscendingNode	247.8337901

```

Comment:
Too much precision for what the source data allows?

Reply:
Fixed; they are now 134 and 248 respectively.